rustler_tests/native/deprecated_macros/Cargo.toml

```diff
@@ -11,3 +11,5 @@ crate-type = ["cdylib"]
 
 [dependencies]
 rustler = { path = "../../../rustler" }
+
+[workspace]
```

Comment:
This is only a crate, but not a workspace. I think this is not needed.

Reply:
I added this to stop it from failing to think it was in a workspace. Based on your input above, I will remove this and add to the root workspace. :+1: